.github/workflows/tests.yml

```diff
@@ -19,15 +19,15 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
     - name: run bats tests
       run: docker run -t -v "${PWD}:/code" bats/bats:latest test/*.bats
   test-stack:
     strategy:
       matrix:
-        stack: [heroku-18, heroku-20, heroku-22]
+        stack: [heroku-18, heroku-20, heroku-22, heroku-24]
```

Comment:
(for another PR) Heroku-18 is EOL and can't be used by customers any more, so support for it could (and probably should to clean things up) be dropped from the buildpack. I've filed #190 for that - would you have a moment to take that one too? :-)

Reply:
sure thing